### ISSUE TITLE
Header level embedding, like done by obsidians embeds.

### ISSUE THREAD

**Issue by AlyceOsbourne** [mProjectsCode/obsidian-meta-bind-plugin#268]
### Is your Feature Request Related to a Problem?

I am probably not using this plugin in the most traditional sense. But having this kinda embedding would halve the number of notes I need for this system, or more. 

### Describe the Feature you'd Like

"Simply" emulating the built in embedding, but as a meta bind embed. THis would mainly include header level embeds, like `![[Note#Header]]` The goal being to extend the functionality.

**Comment by redbeardymcgee** [comment]
Not intending to dogpile, but block level embeds would also be fantastic. I don't think it warrants a separate feature request right now.

`[[Note^block-id]]`

**Comment by DryIce1** [comment]
Hi, 

I recycle my buttons by creating a buttons index note which I embed in pages that use any buttons e.g. each of my periodic notes have a few buttons each, some are reused. This is very convenient because there is only one place to edit buttons, and that place updates any buttons elsewhere in the vault that are referencing.

``` meta-bind-embed
[[Meta bind button index - Meta bind embed]]
```

Similarly, I would love to recycle meta bind selectors by organising them under headers in an index note and referencing that header with a meta-bind-embed block.

e.g. 
``` meta-bind-embed
[[Meta bind datascope - embed#Select type]]
```
Would be able to reuse and maintain order for the following throughout my dataview media-type notes:

## select **status**
```meta-bind
INPUT[select(
option(reading),
option(not started),
option(next),
option(active), 
option(done), 
option(waiting), 
option(cancelled)
):statusBind]
```

awesome plugin, thank you

**Comment by Mistmage** [comment]
+1 for that, as it allows to only embed a heading, allowing to hide a bunch  of stuff, like comments other versions and so on, under a different heading.

**Comment by soggymuse** [comment]
Ooh yes, another +1 for this (header and block/section embed).

**Comment by Sirnii** [comment]
+1 would be very useful

**Comment by Xelbayria** [comment]
Good idea but one feedback to improve the `meta-bind-embed`. Meta-Bind is the only one plugin that is capable of what I want it to do. 

here's the image: 
<details><summary>Image</summary>
<p>

<img width="886" height="303" alt="Image" src="https://github.com/user-attachments/assets/e03e5716-c870-4d9f-b4b8-abb0c3ea0caa" />

</p>
</details> 

I've tried to use `meta-bind-embed`, it almost work like a charm but it doesn't retain the multi-column. alas! It would be good to see if the `meta-bind-embed` can 100% mirror to how the base note is structured including multi-column. 

Fun is that I can reuse the "template" note like you see in the picture for books. If i want to make a small adjustment. i can do that in "template" note without doing other 100 books. save me a lot of time. 

I'm surprised that such a feature is not seen in any plugins or anywhere.
